lifetiles-graph/src/main/java/nl/tudelft/lifetiles/annotation/model/ResistanceAnnotation.java

```diff
@@ -71,35 +71,35 @@ public ResistanceAnnotation(final String geneName,
     /**
      * @return the drugResistance
      */
-    public String getDrugResistance() {
+    public final String getDrugResistance() {
```

Comment:
That would prevent future extensions to the class, instead of just preventing the methods from being overridden.